web/models/package.ex

```diff
@@ -50,7 +50,7 @@ defmodule HexWeb.Package do
     changeset(package, :update, params)
   end
 
-  def is_owner(package, user) do
+  def owner?(package, user) do
```

Comment:
We should keep the old name because functions ending with ? signals a boolean return value.